docs/sources/helm-charts/mimir-distributed/run-production-environment-with-helm/configuration-with-helm.md

```diff
@@ -10,7 +10,7 @@ aliases:
 
 # Manage the configuration of Grafana Mimir with Helm
 
-The `mimir-distributed` Helm chart provides interfaces to set Grafana Mimir [configuration parameters](/docs/mimir/v2.5.x/operators-guide/configure/reference-configuration-parameters/) and customize how Grafana Mimir is deployed on a Kubernetes cluster. This document describes the configuration parameters.
+The `mimir-distributed` Helm chart provides interfaces to set Grafana Mimir [configuration parameters](/docs/mimir/v2.5.x/operators-guide/configure/references-configuration-parameters/) and customize how Grafana Mimir is deployed on a Kubernetes cluster. This document describes the configuration parameters.
```

Comment:
Shouldn't be like this?

```suggestion
The `mimir-distributed` Helm chart provides interfaces to set Grafana Mimir [configuration parameters](/docs/mimir/v2.5.x/operators-guide/configure/references/configuration-parameters/) and customize how Grafana Mimir is deployed on a Kubernetes cluster. This document describes the configuration parameters.
```

Reply:
I do not think so because it is a static link to a specific release. @krajorama, this is what we decided, right?

Reply:
Yes, we've decided to link the correct Mimir/GEM documentation from the Helm chart docs. Which means the version that the chart actually contains. Hopefully this will be done with some version variable that we can set in one place sometime in the not so distant future.
Given this, the v2.5.x documentation is in the right place already, no need to change in this PR I think.